README.md

```diff
@@ -28,7 +28,7 @@ What is the purpose of the `alt` attribute? e.g:
 ```
 
 ```text
-Your answer here.
+For 508 compliance, using an 'alt' tag is for people with disabilities.
```

Comment:
:+1:
Yes.  Why?  What does it do?  (I don't need  reply.  I just want to ensure you know.)